Fix http output in the page (#1119)
aiven/devportal: docs/tools/api/examples.rst

API examples
============

> Here are a few examples (using curl) to get you started with the Aiven API. Replace all ``TOKEN`` with your own value.

Reviewer: ```suggestion
Here is an example to get you started with the Aiven API using curl. Replace ``{TOKEN}`` with your own value of the authentication token.
```

List of cloud regions
---------------------

.. code::

>   curl -H "Authorization: Bearer TOKEN" https://api.aiven.io/v1/clouds

Reviewer: ```suggestion
  curl -H "Authorization: Bearer {TOKEN}" https://api.aiven.io/v1/clouds
```
